resources/views/model.blade.php

```diff
@@ -67,41 +70,37 @@ class {{ $modelBaseName }} extends Model
 
     ];
     @endif
-
-    @if ($translatable->count() > 0)// these attributes are translatable
+@if ($translatable->count() > 0)// these attributes are translatable
     public $translatable = [
     @foreach($translatable as $translatableField)
     "{{ $translatableField }}",
     @endforeach
 
     ];
     @endif
-
-    @if (!$timestamps)public $timestamps = false;
+@if (!$timestamps)public $timestamps = false;
     @endif
 
     protected $appends = ['resource_url'];
 
     /* ************************ ACCESSOR ************************* */
 
-    public function getResourceUrlAttribute() {
+    public function getResourceUrlAttribute()
```

Comment:
It will be great if the resource could be called admin_resource_url, but it means to change it everywhere. I am not sure with compatibility issue then, but still it is just generator.